src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -9,6 +9,7 @@ CA1047 | Design | Info | DoNotDeclareProtectedMembersInSealedTypes, [Documentati
 CA1069 | Design | Info | EnumShouldNotHaveDuplicatedValues, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1069)
 CA1700 | Naming | Disabled | DoNotNameEnumValuesReserved, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1700)
 CA1830 | Performance | Info | PreferTypedStringBuilderAppendOverloads, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1830)
+CA1834 | Performance | Info | PreferStreamAsyncMemoryOverloads, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1834)
```

Comment:
This was the latest ruleID available at the time of creating my branch.
Do we have a mechanism to reserve a ruleID while you work on it?